roles/timer/templates/service

```diff
@@ -0,0 +1,10 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description={{ timer_descr | default(timer_name + " Timer") }}
```

Comment:
please put defaults into `roles/timer/default/main.yml`
so debugging of values is easiert

Reply:
There are the defaults.

Reply:
done